tests/unit/test_provider.py

```diff
@@ -126,13 +130,13 @@ def test__on_logging_relation_changed(self, mock_unit_ip, *unused):
                 "DEBUG:charms.loki_k8s.v0.loki_push_api:Saved alerts rules to disk",
             )
 
-    @patch("ops.testing._TestingPebbleClient.make_dir")
-    @patch("os.makedirs")
-    @patch("ops.testing._TestingPebbleClient.remove_path")
+    @patch("ops.testing._TestingPebbleClient.make_dir", Mock())
+    @patch("os.makedirs", Mock())
+    @patch("ops.testing._TestingPebbleClient.remove_path", Mock())
```

Comment:
Can the test be updated to work without these mocks?

Reply:
Yes, except for `makedirs`